.github/workflows/plantuml.yml

```diff
@@ -20,6 +20,9 @@ jobs:
         uses: actions/checkout@v3
         with:
           fetch-depth: 0
+          # https://github.com/stefanzweifel/git-auto-commit-action#workflow-should-run-in-base-repository
```

Comment:
This anchor point to nowhere

Reply:
that anchor is inside an accordion which is collapsed by default, will link to the parent 🤦🏻